.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -98,7 +98,8 @@ jobs:
         conf: ['', '--enable-abi-compat', '--enable-deprecated --enable-debug=full',
                '--enable-dpdk-zero-copy --disable-static-applications',
                '--disable-host-optimization', '--disable-host-optimization --enable-abi-compat',
-               '--without-openssl --without-pcap']
+               '--without-openssl --without-pcap', 'CFLAGS="-march=armv8.2-a"',
+               'CFLAGS="-march=armv8-a+lse"']
```

Comment:
`-O2` flag, which revealed earlier an implementation bug, should also be used. It could be enabled with a following syntax: https://github.com/OpenDataPlane/odp/commit/f4ddd28b96b9dd6e7a0a1ed7362065e5fe96099b

Reply:
Sorry, my bad. I applied the older patch when upstreaming. I have updated the PR. Thanks Matias!